flight/mcu/src/mcu_ac/main.cpp

```diff
@@ -6,7 +6,7 @@
  *
  * Active Control Program
```

Comment:
you can remove this line. This does a lot more than active controls now. Also we put this as "Active Control Program" because we going to make a copy of this for the other teensy called "Hybrid Program" which handled hybrid engine avionics.